doc/ko/uftrace-graph.md

```diff
@@ -91,6 +95,8 @@ GRAPH 옵션
 :   타입(TYPE)으로 일치하는 패턴을 보여준다. 가능한 형태는 `regex`와 `glob`이다.
     기본 설정은 `regex`이다.
 
+\--with-syms=*DIR*
+:   DIR 디렉토리의 .sym 파일에서 심볼(symbol) 데이터를 읽는다. 이는 심볼(symbol) 데이터가 제거된 바이너리 파일을 다루는데 유용하다. 바이너리 파일 이름은 저장할 때와 사용할 때 동일해야 한다.
```

Comment:
Please break the long line (around 80 characters).

Also please check the other man pages for the same option.  I'd like to have them with the same translation.

Reply:
Ok! I'll take it.